dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1115,9 +1120,12 @@ private void wesStatus(WorkflowExecutionServiceApi clientWorkflowExecutionServic
     private void wesRunLogs(WorkflowExecutionServiceApi clientWorkflowExecutionServiceApi, final String workflowId) {
         try {
             RunLog response = clientWorkflowExecutionServiceApi.getRunLog(workflowId);
-            out(response.toString());
+            ObjectMapper mapper = new ObjectMapper();
```

Comment:
Just being paranoid, if a WES server were compromised and sent us an evil response, could this lead to code being executed?

I don't think so (it was a YAML parser that had an exploit, which has been patched), but like I said, being paranoid.

Added @david4096 as a reviewer.

Reply:
Took a look, you can do type overrides in Jackson like this:

```
{
  "@class":"MyApp.Obj",
  val:[
    "java.lang.Long",
    1
  ]
}
```

However, it appears the exploit requires you explicitly turn on `mapper.enableDefaultTyping`. I think we are safe based on my current understanding.

https://adamcaudill.com/2017/10/04/exploiting-jackson-rce-cve-2017-7525/